Library/Homebrew/keg.rb

```diff
@@ -483,6 +483,8 @@ def link(verbose: false, dry_run: false, overwrite: false)
     ObserverPathnameExtension.n
   end
 
+  def dsymutil; end
```

Comment:
This name feels a bit too macOS specific for something that someone may want to implement on Linux.

Reply:
Done in e46a61e181f8e324d1bc4222bedf692c11ad84fa